stale_unassigned_rule.py

```diff
@@ -20,7 +20,7 @@
 import logging
 
 
-class StaleMajorOrAboveRule(FlinkJiraRule):
+class StaleUnassignedRule(FlinkJiraRule):
     """
     Tickets major and above without a fixVersion need an assignee, or an update within {stale_<blocker|critical|major>.stale_days},
```

Comment:
nit: outdated comment `Tickets major and above`.

Reply:
Fixed.